src/handlers/miner_hbbft_handler.erl

```diff
@@ -542,9 +551,65 @@ state_sigs_input(SigsIn, #state{hbbft=HBBFT}=S0) ->
         {{some, {done, _}}, S3} ->
             {pending, S3};
         {none, S3} ->
-            {pending, S3}
+            case state_sigs_maybe_switch_to_varless(S3) of
+                {{some, SigsVarless}, S4} ->
+                    %% TODO Q: Keep this log line? Shorten it? Rephrase?
+                    lager:warning(
+                        "Autoskipped to varless. New sigs: ~p",
```

Comment:
I would say we're mostly interested in *who* did the signatures that tripped us over to varless, not their signatures. You could even use the sig_position function to simply get an index list.